lib/matplotlib/backends/backend_agg.py

```diff
@@ -48,6 +48,9 @@
 except ImportError:
     _has_pil = False
 
+__all__ = list(map(str, ['backend_version', 'new_figure_manager',
```

Comment:
I would not include the backed-specific canvas in the all, just the `FigureCanvas`